src/components/input/math-input.js

```diff
@@ -485,10 +504,15 @@ class MathInput extends React.Component {
         }
 
         if (y > this._containerBounds.bottom) {
-            y = this._containerBounds.bottom;
+            y = this._containerBounds.bottom - 10;
```

Comment:
Is this 10 the cursor width? Or is it padding?

Reply:
Great question! It's accounting for the padding around the values. I'll add a comment.